### PR TITLE
Caution not to connect the servo's Output Enable pin

### PR DESCRIPTION
**Pull request by BrianHenryIE**
The kit I purchased from Donkeycar had five cables, so I connected the five sequential pins. When I drove my car, it quickly cut out.

Also a quick grammar fix.

### CODE REVIEW

File: docs/guide/build_hardware.md [hunk 1]
```diff
@@ -107,7 +107,9 @@ Plug in the Servo driver the same as the Raspberry Pi, just keep in mind that th
 
 ![Jetson Servo](/assets/Servo_Wiring.png)
 
-Finally this is the Donkey Assembled.  
+Finally this is the Donkey assembled.  
+
+Although it may seem reasonable to connect all pins in the set of six from the servo, the second pin is *OE Output enable. ... When the pin is high the outputs are disabled. Pulled low by default so it's an optional pin!* Connecting this pin can result in your car unexpectedly stopping.
```
Comment (nvtkaszpir) [inline docs/guide/build_hardware.md]: Maybe move it just below the line 106?

**Comment by BrianHenryIE** [comment]
Good call @nvtkaszpir. I was reading the docs for another reason earlier and when I saw the picture I thought to add the warning. I admit, I didn't read the page through for clarity.

File: docs/guide/build_hardware.md (hunk 1 above)
Comment (tawnkramer) [reply docs/guide/build_hardware.md]: there's only 3 pins from the servo. I don't get it.

Comment (BrianHenryIE) [reply docs/guide/build_hardware.md]: http://docs.donkeycar.com/assets/Servo_Wiring.png

It's these close six pins I was referring to. I guess I meant "servo driver". Unless there's a clearer term to use..?

Comment (nvtkaszpir) [reply docs/guide/build_hardware.md]: @BrianHenryIE you mean 6 pins on PCA controller (there are cables connected to it 0 black, gray, white, violet)?

**Comment by tawnkramer** [comment]
These are I2C interface pins. Not servo or PCA.

File: docs/guide/build_hardware.md (hunk 1 above)
Comment (BrianHenryIE) [reply docs/guide/build_hardware.md]: @nvtkaszpir Yes, the six pins on the PCA controller (GND, OE, SCL, SDA, VCC, V+). If OE is connected the car will shut off unexpectedly. The kit I bought from [DonkeyCar.com](https://store.donkeycar.com/) had a set of five joined wires/connectors so I just connected them all. 

@tawnkramer It's the pins on the PCA9865 that connect to my Jetson Nano I'm referring to.  The photo isn't mine but displays things properly.

**Comment by DocGarbanzo** [comment]
I'll close this for now, I don't think "it may seem reasonable to connect all pins" of the PCA9865 - this is a standard I2C interface w/ +,-, SCL, SDA, no-one seems to have problems with that so far.

**Comment by BrianHenryIE** [comment]
> no-one seems to have problems with that so far.

I had a problem with this. 

When the kit comes with five cables in one piece, the [affordance](https://www.merriam-webster.com/dictionary/affordance) is to connect all five.

**Pull request closed without merging**